manifests/y/YmSoft/Nalgaeset/10.4/YmSoft.Nalgaeset.installer.yaml

```
@@ -0,0 +1,17 @@
+PackageIdentifier: YmSoft.Nalgaeset
+PackageVersion: 10.4
+Installers:
```

```suggestion
Scope: machine
Installers:
```
`C:\Program Files\YmSoft\NgsEdit`